docs/2.8.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -194,6 +194,64 @@ The ping command creates two contracts between the admin parties, then exercises
 Version Specific Notes
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Upgrade to Release 2.8
+^^^^^^^^^^^^^^^^^^^^^^
+
+Version 2.8 extends the database schema. If you use the "migrate and start" feature, the database schema will be automatically updated.
+Otherwise, perform the manual database migration steps outlined in the :ref:`database migration steps <migrating_the_database>`.
+
+Explicit contract disclosure (Beta)
+"""""""""""""""""""""""""""""""""""
+The explicit contract disclosure feature is now in Beta and is enabled by default.
+Refer to the explicit contract disclosure :ref:`documentation <explicit-contract-disclosure>` for details.
+Note that the ``participant.ledger-api.explicit-disclosure-unsafe`` feature flag is deprecated and replaced by
+``participant.ledger-api.enable-explicit-disclosure``, set to true by default.
+
+Breaking configuration changes
+""""""""""""""""""""""""""""""
+
+**KMS wrapper-key configuration value**
+The configuration value for the KMS wrapper-key now accepts a simple string.
+Update your configuration as follows:
+
+.. code:: bash
+
+    crypto.private-key-store.encryption.wrapper-key-id = { str = "..."} # version 2.7
+    crypto.private-key-store.encryption.wrapper-key-id = "..." # version 2.8
```

Comment:
@JoaoSa-da using @thibault-da machinery, could we easily make this change backward compatible ?

Reply:
@rgugliel-da what is @thibault-da machinery? :)

Reply:
Not with the existing machinery, it allows to keep changes to configuration _paths_ backwards compatible but not configuration _types_ 
I thought we had a discussion about this @JoaoSa-da and said it didn't need to be no? Don't remember why though. We can still make it backwards compatible with some `ConfigReader` magic though, I'll have a look today actually